custom/icds_reports/tests/test_aggregation_script.py

```diff
@@ -553,7 +555,7 @@ def setUpClass(cls):
         sup2.metadata = {"is_test_location": "test"}
         sup2.save()
 
-        cls.helper = LocationAggregationHelper()
+        cls.helper = LocationAggregationDistributedHelper()
```

Comment:
can we set this conditionally based on `settings.ICDS_USE_CITUS` or similar?

Reply:
We can, but the distributed version is guaranteed to work on a non-citus db, so it does not buy us much (I am a little surprised the original implementation did not use that one already)

Reply:
Oh neat, I didn't know that.